.github/workflows/code_review.yml

```diff
@@ -12,9 +12,10 @@
       - name: Checkout repository
         uses: actions/checkout@v3
       - name: Code Review
-        uses: freeedcom/ai-codereviewer@main
+        uses: ./
         with:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          OPENAI_API_KEY: ${{ secrets.OPENAI_API_KEY }}
-          OPENAI_API_MODEL: "gpt-4-1106-preview"
+          INFERENCE_API_KEY: ${{ secrets.INFERENCE_API_KEY }}
+          MODEL: "gpt-4o-mini"
           exclude: "yarn.lock,dist/**"
+          custom_instructions: "Ignore logging changes, Only focus on important changes"
```

Comment:
The `custom_instructions` parameter should be carefully reviewed to ensure that it does not inadvertently exclude important changes. The instructions should be clear and precise to avoid missing critical feedback during the code review process.